untie/src/main.cpp

```diff
@@ -5,7 +5,8 @@
 using namespace knot;
 
 int main() {
-    log::debug("Hi there");
```

Comment:
unsure why dog::debug is above the setup does this still work ?

untie/main.cpp is out of date currently manages untie object that manages the window submodule

Reply:
The commits removes this line of code and replaces it with the setup.